packages/frontend/src/components/account/CreateHouseModal.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Input, Modal, Text } from '@nextui-org/react';
+import React, { useState } from 'react';
+import { HouseServices } from '../../services/HouseService';
+
+interface CreateHouseModalProps {
+  createVisible: boolean;
+  setCreateVisible(value: boolean): void;
+}
+const CreateHouseModal: React.FC<CreateHouseModalProps> = ({
+  createVisible,
+  setCreateVisible,
+}) => {
+  interface NewHouseDetails {
+    phone: string;
+    address: string;
+    name: string;
+  }
+
+  const [newHouseouseDetails, setNewHouseDetails] = useState<NewHouseDetails>({
+    phone: '',
+    address: '',
+    name: '',
+  });
+
+  const closeCreateHandler = () => {
+    setCreateVisible(false);
+  };
+
+  async function handleCreatingHouse() {
+    try {
+      setCreateVisible(false);
+      const response = await HouseServices.createHouse(newHouseouseDetails);
```

Comment:
This response isn't needed. I think you just need to navigate to `/home` after the `await` (right now after creating nothing happens).